### ISSUE TITLE
Deployment failed to un-tar bundle

### ISSUE THREAD

**Issue by dnszero** [jkrup/meteor-now#32]
From the `now` output once the deployment URL is available:
```
gzip: stdin: invalid compressed data--format violated
[0m[91mtar: Child returned status 1
[0m[91mtar: Error is not recoverable: exiting now
The command '/bin/sh -c tar -xzf bundle.tar.gz' returned a non-zero code: 2
```

This happens when deploying from Mac and Windows.

When I manually `cat *sf-part* > bundle.tar.gz` in .meteor/local/builds, the resulting bundle also fails with a similar error on my local machines:
```
<...snip...>
x bundle/programs/server/npm/node_modules/fs.realpath/LICENSE
x bundle/programs/server/npm/node_modules/fs.realpath/README.md
tar: gzip decompression failed
tar: Error exit delayed from previous errors.
```

However, if I manually run the build command, and DO NOT SPLIT THE FILE, I'm able to untar ungzip it with no errors.

The problem is definitely occurring as a result of the split-file process (or as a result of cat'ing them back together). The resulting error happens on Mac, Windows 10, and on Zeit Now.

I don't have a publicly available repo to share yet. If it helps any the resulting bundle.tar.gz is 12924604 bytes (which was split into 13 parts).

**Comment by purplecones** [comment]
@dnszero thanks for submitting this. We identified the problem and have a fix in the works. Should get resolved soon.

**Comment by jkrup** [comment]
Tahnks for giving us this much diagnostic info @dnszero . The "13 parts" cue'd us into the issue. The new `split-file` is numeric rather than alphabetical so when you have >9 parts when running `cat *` it goes `part1`, `part10`,..`part2`, `part3`  (the wrong order).

**Comment by jkrup** [comment]
We just pushed an update that should fix this please `npm install -g meteor-now@0.2.1` to get the latest version and re-run your deploy command!

**Comment by dnszero** [comment]
Thanks guys! And thanks for your hard work on meteor-now. It's helped inspire me to get started with meteor again :-)
